.github/workflows/submitty_ci.yml

```diff
@@ -600,6 +520,76 @@ jobs:
           - name: Check for site errors
             working-directory: ${{ env.SUBMITTY_REPOSITORY }}
             run: sudo bash tests/test_site_error_log.sh
```

Comment:
This needs to be run after every job.  Perhaps there is a way to do this automatically by adding something like [this](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runspost) to the setup action you created.